src/main/java/com/google/neighborgood/task/Task.java

```diff
@@ -143,4 +139,8 @@ private void setIsOwnerCurrentUser(String ownerId) {
     if (userId.equals(ownerId)) this.isOwnerCurrentUser = true;
     else this.isOwnerCurrentUser = false;
   }
+
+  public String getOverview() {
```

Comment:
Is the getter here necessary? I don't think I saw any new changes depending on this function. If it is not necessary, I suggest deleting the getter here because getter is "evil".

Reply:
I added this for a test that I was implementing that I ended up deleting, but forgot to deleting this getter. Will update in the new commit!